### ISSUE TITLE
Crash unexpectedly on iPad

### ISSUE THREAD

**Issue by digitalbottle** [DagAgren/toot#490]
System info:
iPad pro 10.5 inch
ios 14.4 & 14.4.1
Thx.

**Comment by DagAgren** [comment]
I need some more information about this. Is that the latest Toot version, and what were you doing when this happened?

**Comment by digitalbottle** [comment]
Thx for the immediate reply, however I'm not so sure what caused this issue; I'd love to find it by testing repeatedly but once I crash out, I cannot go back anymore. There's nothing I can do except uninstall and reinstall. I'll try to remember what happened next time.

**Comment by DagAgren** [comment]
Do you get a crash again on restarting?

**Comment by digitalbottle** [comment]
Yep, I cannot restart it, it will crash again afterwards, repeatedly (not by accident) every time when I tap to restart. I reinstalled twice and met the same issue, and I'm waiting for the third time.

**Comment by DagAgren** [comment]
Is it the latest version of Toot?

**Comment by digitalbottle** [comment]
Yes, it is.

**Comment by Canorus** [comment]
Same issue on iPadOS 14.6. It happened when I tried to open DM tab from sidebar. I reinstalled from App Store(which implies it's the latest version as of writing), opened DM tab, app crashed and keep crashing on restart.
